pkg/execution/plugins/atlassian/iamrole/iam_role_manager_test.go

```diff
@@ -89,9 +90,21 @@ func TestGenerateRoleInstance(t *testing.T) {
 		ServiceName:     "test-svc-app",
 		OAPResourceName: "app-iamrole",
 		ServiceEnvironment: oap.ServiceEnvironment{
-			NotificationEmail:            "an_owner@example.com",
-			LowPriorityPagerdutyEndpoint: "https://events.pagerduty.com/adapter/cloudwatch_sns/v1/123",
-			PagerdutyEndpoint:            "https://events.pagerduty.com/adapter/cloudwatch_sns/v1/456",
+			NotificationEmail: "an_owner@example.com",
+			AlarmEndpoints: []rds.MicrosAlarmSpec{
```

Comment:
we should probably find a better place for `MicrosAlarmSpec` but i failed to find one, so open for suggestions. Maybe `types.go` in a wiring root dir?

Reply:
See related @ash2k suggestion on cross wiring dependencies https://github.com/atlassian/voyager/pull/54#discussion_r247714217  maybe wiring/api ?

Reply:
`wiring` package should not know anything about particular autowiring plugins. I think we can move this type to `pkg/orchestration/wiring/wiringutil/oap/types.go`